CREDITS.md

```diff
@@ -10,7 +10,7 @@ The following individuals are responsible for curating the list of issues, respo
s://github.com/Luc45), [Thien Nguyen (@tatthien-zz)](https://github.com/tatthien-zz), [Shiva Poudel (@shivapoudel)](https://github.com/shivapoudel), [null (@nextgenthemes)](https://github.com/nextgenthemes), [Tom Usborne (@tomusborne)](https://github.com/tomusborne), [dean shmuel (@deanshmuel)](https://github.com/deanshmuel), [Peter Adams (@padams)](https://github.com/padams), [Kevin Batdorf (@KevinBatdorf)](https://github.com/KevinBatdorf), [Q (@qstudio)](https://github.com/qstudio), [null (@om4csaba)](https://github.com/om4csaba), [Roman Sapezhko (@shmidtelson)](https://github.com/shmidtelson), [null (@luizkim)](https://github.com/luizkim), [René Hermenau (@rene-hermenau)](https://github.com/rene-hermenau), [Lewis Cowles (@Lewiscowles1986)](https://github.com/Lewiscowles1986), [Dominik Schilling (@ocean90)](https://github.com/ocean90), [Santiago Becerra (@sanbec)](https://github.com/sanbec), [Marijn Bent (@marijnbent)](https://github.com/marijnbent), [Jasan (@jasan-s)](https://github.com/jasan-s), [Dale Nguyen (@dalenguyen)](https://github.com/dalenguyen), [Darren Cooney (@dcooney)](https://github.com/dcooney), [Karolína Vyskočilová (@vyskoczilova)](https://github.com/vyskoczilova), [Rafał Sztwiorok (@sztwiorok)](https://github.com/sztwiorok), [Alec Rust (@AlecRust)](https://github.com/AlecRust), [Jonny Harris (@spacedmonkey)](https://github.com/spacedmonkey), [Doeke Norg (@doekenorg)](https://github.com/doekenorg), [Andrew Heberle (@andrewheberle)](https://github.com/andrewheberle), [Takashi Hosoya (@tkc49)](https://github.com/tkc49), [Fabian Marz (@fabianmarz)](https://github.com/fabianmarz), [David Herron (@robogeek)](https://github.com/robogeek), [Sergey Kotlov (@sery0ga)](https://github.com/sery0ga), [Samuel Wood (@Otto42)](https://github.com/Otto42), [IgorChernenko (@igorchernenko92)](https://github.com/igorchernenko92), [Richard Muvirimi (@richard-muvirimi)](https://github.com/richard-muvirimi), [Dani Llewellyn (@diddledani)](https://github.com/diddledani), [Peter Wilson (@peterwilsoncc)](https://github.com/peterwilsoncc), [Mukesh Panchal (@mukeshpanchal27)](https://github.com/mukeshpanchal27), [Joe McGill (@joemcgill)](https://github.com/joemcgill),  [Felix Arntz (@felixarntz)](https://github.com/felixarntz),[Evan Mattson (@aaemnnosttv)](https://github.com/aaemnnosttv), [Darin Kotter (@dkotter)](https://github.com/dkotter), [Stephanie Wells (@stephywells)](https://github.com/stephywells), [Siddharth Thevaril (@Sidsector9)](https://github.com/Sidsector9), [Jason Adams (@JasonTheAdams)](https://github.com/JasonTheAdams), [Dharmesh Patel (@iamdharmesh)](https://github.com/iamdharmesh), [Uriahs Victor (@UVLabs)](https://github.com/UVLabs), [Simon Dowdles (@10upsimon)](https://github.com/10upsimon), [Volodymyr Kolesnykov (@sjinks)](https://github.com/sjinks), [Max Lyuchin (@cadic)](https://github.com/cadic), [Rahul Prajapati (@rahulsprajapati)](https://github.com/rahulsprajapati), [Faisal Alvi (@faisal-alvi)](https://github.com/faisal-alvi).
```

Comment:
Really minor and can be ignored:

- An extra space before `[Felix Arntz` can be removed.
- No spaces before `[Evan Mattson`, should be added one.